src/Stripe.net/Services/Account/AccountCreateOptions.cs

```diff
@@ -1,15 +1,114 @@
 namespace Stripe
 {
+    using System.Collections.Generic;
     using Newtonsoft.Json;
+    using Stripe.Infrastructure;
 
-    public class AccountCreateOptions : AccountSharedOptions
+    public class AccountCreateOptions : BaseOptions, IHasMetadata
     {
+        [JsonProperty("account_token")]
+        public string AccountToken { get; set; }
+
+        /// <summary>
+        /// Optional information related to the business.
+        /// </summary>
+        [JsonProperty("business_profile")]
+        public AccountBusinessProfileOptions BusinessProfile { get; set; }
+
+        /// <summary>
+        /// The business type. One of <c>individual</c> or <c>company</c>.
+        /// </summary>
+        [JsonProperty("business_type")]
+        public string BusinessType { get; set; }
+
+        /// <summary>
+        /// Information about the company or business. This field is null
+        /// unless <c>BusinessType</c> is set to <c>company</c>.
```

Comment:
It's not `<c>` when you reference another field. Instead, it's `<see cref="BusinessType"/>` for example so that the IDE can reference the other field.